baseplate/clients/sqlalchemy.py

```diff
@@ -61,7 +67,10 @@ def engine_from_config(
         url.username = credentials.username
         url.password = credentials.password
 
-    return create_engine(url, **kwargs)
+    if options.pool_recycle:
```

Comment:
Instead of having two whole calls to `create_engine`, it'd be cleaner to follow the pattern used in other clients where we just `setdefault()` on `kwargs`.

https://github.com/reddit/baseplate.py/blob/master/baseplate/clients/cassandra.py#L77-L78

Reply:
way cleaner!